submissions/DmitryHniezdilov/task-js-dom/js/main.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+import { dataList, defaultContentItem } from "./data.js";
+
+const elemLogo = document.querySelector(".js-elem-logo");
+const elemMenu = document.querySelector(".js-elem-menu");
+const elemContent = document.querySelector(".js-elem-content");
+
+const createMenuMarkup = (data) => {
+  const menuListItems = data.map((item) => {
+    return `
+                <li class="menu__item" >
+                    <button class="menu__btn" type="button" data-item-id="${item["id"]}">${item["Title"]}</button>
+                </li>
+            `;
+  });
+
+  return `
+            <ul class="menu__list">
+                ${menuListItems}
+            </ul>
+        `;
+};
+
+const changeMinToHm = (min) => {
+  const MIN = Number(min.split(" ")[0]);
+  let h = Math.floor(MIN / 60);
+  let m = Math.abs(MIN % 60);
```

Comment:
Variable names should not be too short or too obscure: (although some conventional shorthands are OK; e.g. i when looping through array by index, x/y to denote coordinates, and other similar cases)

Reply:
Done